### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 10th June 2023 @ 15:00 UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#843]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://forum.monero.space/d/79-how-to-join-the-monero-core-team-matrix-server-web)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero Moon](https://www.themoneromoon.com/)       
    - Monerokon: Tables available for people who want to showcase or even sell things for Monero
    - Free 7$ in crypto if you like/retweet this [MoneroKon tweet](https://twitter.com/MajesticBank/status/1663716497380737024) - MajesticBank
    - [Add monero-node to Umbrel-OS](https://github.com/getumbrel/umbrel-apps/pull/617) pull request - [m1m3-50](https://github.com/m1m3-50)
    - [successonly.work to end](https://www.reddit.com/r/Monero/comments/13z98ke/comment/jmq7mz4/?utm_source=share&utm_medium=web2x&context=3) - update from spirobel
    - [Tor releases a PoW ddos mitigation](https://www.reddit.com/r/Monero/comments/13zc1qn/comment/jmqiqlj/?utm_source=share&utm_medium=web2x&context=3) - summary by hinto 
    - [New XMRig release 6.19.3](https://github.com/xmrig/xmrig/releases/tag/v6.19.3)
    - [Atomic Wallet rekt?](https://nitter.it/zachxbt/status/1665080799253733377)
    - [mobile mining rack built from KINEX](https://nitter.it/theretromike/status/1665057282332192768)
    - [RINO.io twitter raffle](https://twitter.com/RINOwallet/status/1664072854818762766) - draw every Friday, 100 eur in xmr
    - List of wallets who have informed the community that they've updated to v0.18.2.2
            - [Anonero](anonero5wmhraxqsvzq2ncgptq6gq45qoto6fnkfwughfl4gbt44swad.onion)
            - [CakeWallet](https://cakewallet.com/) + [Monero dot com](https://monero.com/)
            - [EliteWallet](https://elitewallet.sc/)
            - [FeatherWallet](https://featherwallet.org) - will now start 1 second faster , 80% reduction - tobtoht [upstream PR](https://github.com/monero-project/monero/pull/8894) sech1
            - [Mysu](https://mysu.dev/)
            - [Monerujo](https://www.monerujo.io/)
    - Monerujo releases PocketChange (experimental) [reddit post](https://www.reddit.com/r/Monerujo/comments/1412gah/update_for_the_fearless_monerujo_v337_pocket/) / [Commit](https://github.com/m2049r/xmrwallet/pull/901/files) 
    - [Amount of re-orgs over time vs hashrate](https://github.com/monero-project/research-lab/issues/102#issuecomment-1577827259) - suggests a reduction after Fluorine Fermi - chaserene
    - [5000+ Trocador trades in CakeWallet](https://www.reddit.com/r/Monero/comments/142un1v/celebrating_5000_trades_with_trocador_in_cake/)
    - [Monero.observer taking a break](https://monero.observer/monero-observer-announcement-on-break/)
    - [PR to add stealth-address-like functionality to bitcoin-core](https://nitter.it/sethforprivacy/status/1665722022758940672) - SethForPrivacy
    - [MAGIC Monero Fund raising funds to prevent the EAE attack](https://www.reddit.com/r/Monero/comments/144d6vo/magic_monero_fund_raising_funds_to_prevent_the/)
    - [BCH<->XMR Atomic swap method](https://www.reddit.com/r/btc/comments/1455rxe/bchxmr_atomic_swaps_solution_using_introspection/) - shared by Rucknium
5. [CCS updates](https://ccs.getmonero.org/)    
  a. [VostoEmisio Animated Core Concept Videos](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/387) -> voted for merge last meeting (Promises and assurances have been added)
  b. [Wallet SDK for Android](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388) -> More feedback has been obtained. Positive sentiment 
  c. [Resolve abandoned outreach CCS](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/70#note_21324) - Ajs -> On hold
  d. [jeffro256 full-time dev 2023Q3](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/390)    
7. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2023](https://github.com/MoneroKon/)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
8. Open ideas time    
9. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/838)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
```
11:04:59 <plowsof11> Meeting today in ~4 hours https://github.com/monero-project/meta/issues/843 (how have you
been? merge everything, monero events is poppin - also has a meeting today #monero-events:monero.social )

14:32:55 <chowbungaman[m]> Join us in a half hour live
: https://twitter.com/monerotopia/status/1667220477792665601?s=46&t=aPPXwiXRLbGeU5QSb-NdPg Watch live on
YouTube or Twitter or listen to the live spaces. You can also jump up on stage to voice your monero related
opinions via the streamyard link or the spaces. Shows are live Every Sat Morn at 11am est. or catch the
recordings on YouTube, Odyssey , Spotify etc. This weeks special guest is Anhdre

14:32:55 <chowbungaman[m]> talking about the new pocket change feature ! If anyone wants to jump on to say
anything Monerokon please do!

14:54:21 <plowsof11> monerobull: anyone around to share that monerokon has tables available for people to
show/sell things for monero on monerotopia?

15:00:12 <plowsof11> Meeting time https://github.com/monero-project/meta/issues/843

15:00:44 <Rucknium[m]> Hi

15:00:45 <plowsof11> greetings friends

15:00:45 <msvb-lab> Hello.

15:00:53 <BusyBoredom[m]> Hey

15:01:06 <ceetee[m]> hi :)

15:01:09 <hinto[m]> hi

15:02:17 <plowsof11> lets discuss some community highlights, i first just want to share some monero bounties
info, as a few are in the process of being claimed

15:02:22 <plowsof11> [background sync with just the view key](https://github.com/monero-
project/monero/pull/8619) has a 10.5~xmr [bounty](https://bounties.monero.social/posts/41/) - which jberman
says a portion will be shared with reviewers.

15:02:22 <plowsof11> [Tests: Add blake2b Test Cases](https://github.com/monero-project/monero/pull/8901) has a
6.5~xmr [bounty](https://bounties.monero.social/posts/75/) which a new contributor xFFC is in the process of
completing (Koe and selsta busy reviewing)

15:02:22 <plowsof11> tobtoht will be getting 0.71 xmr for retroactively completing this
[bounty](https://bounties.monero.social/posts/86/) for adding themes to featherwallet. (bounties are not for
work already done but the funds where donated before it was marked as completed, some very enthusiastic
donors)

15:02:22 <plowsof11> btc<->xmr atomic swap guides trnalsted into chinese. 1.5 xmr
[bounty](https://bounties.monero.social/posts/5) - 0.3 of which is to be given to a reputable translator,
anyone? (that isnt the person who submitted the doc)

15:05:46 <plowsof11> Rucknium has shared/brought into reality this fund raiser for churning research
https://monerofund.org/projects/eae_attack_and_churning (reddit thread
https://www.reddit.com/r/Monero/comments/144d6vo/magic_monero_fund_raising_funds_to_prevent_the/)

15:08:40 <plowsof11> monero observer is on a break, but the active newsletters this week have been
https://revuo-xmr.com/ and https://localmonero.co/the-monero-standard?language=en. ive also got a few items on
the meeting issue

15:09:42 <Rucknium[m]> There is progress on the BCH<>XMR 10.5 XMR bounty:
https://www.reddit.com/r/btc/comments/1455rxe/bchxmr_atomic_swaps_solution_using_introspection/

15:10:31 <plowsof11> are they still waiting for h4shed to look/respond to that (he works with/for the
farcaster team right?)

15:11:10 <Rucknium[m]> h4ashed and/or kayabaNerve

15:11:36 <Rucknium[m]> Still waiting? It was posted about 24 hours ago

15:12:35 <plowsof11> but i want to know now :'(

15:13:41 <plowsof11> Morpheus added https://moneroj.net/coincards/ (monero usage is increasing against btc)

15:14:56 <plowsof11> i see gupax being mentioned more and more on reddit , and the releases page is active
https://github.com/hinto-janai/gupax/releases nice work hinto

15:16:23 <hinto[m]> someone please fund me an ARM mac

15:16:41 <hinto[m]> testing gupax via VMs and VPSs is painful :D

15:19:59 <plowsof11> maybe they could be included in a monerokon goodybag some day - a funding ccs like
gingers ssd's but for hardware for some contributors seems plausible

15:21:11 <plowsof11> anything else on anyones minds - waiting on invoices for RIAT to move stagnant funds from
a ccs... cryptogrampys hotshop ccs was marked as completed the other day.. any outstanding payments for ccs
payouts pls contact luigi1111 (i think everyone is paid for now)

15:21:19 <msvb-lab> What should be included, SSDs or Macintosh computers? I don't understand.

15:21:28 <BusyBoredom[m]> Slow progress on standalone acceptxmr, haven't gotten to put the time in I want to,
but its not abandoned.

15:22:15 <plowsof11> the irc bridge is playing up - msvb-lab it was just a dream - an entire MAC laptop in a
goodybag is asking alot lol

15:22:36 <Rucknium[m]> OSPEAD is moving forward. I am running processes on over 200 threads.

15:22:39 <msvb-lab> Yes okay, a nice dream yes. Lots of things are possible, realistically.

15:24:34 <plowsof11> thanks for the updates, lets jump into the merge list

15:24:43 <plowsof11>   a. [VostoEmisio Animated Core Concept Videos](https://repo.getmonero.org/monero-
project/ccs-proposals/-/merge_requests/387) -> voted for merge last meeting (Promises and assurances have been
added)

15:25:24 <plowsof11> everyone feel the same way about merging this for funding of 9xmr? i left my own
reasoning in this comment here https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/387#note_21398

15:28:57 <plowsof11> merge it, moving on b. [Wallet SDK for Android](https://repo.getmonero.org/monero-
project/ccs-proposals/-/merge_requests/388) -> More feedback has been obtained. Positive sentiment

15:31:29 <plowsof11> ive had feedback from people wishing to haze valldrac for not delivering the other ccs,
molly, and generally want assurances that monero will be integrated into molly - ive read the ccs, and each
milestone has a verifiable payout criteria / positive feedback and valldrac is interacting well with the
seraphis migration team already

15:33:07 <plowsof11> (expiry dates also, and maintenance built in)

15:33:33 <r4v3r23[m]1> plowsof11: this CCS is not dependent on Molly, and has clear payout criterea. there is
no reason to block it

15:34:00 <r4v3r23[m]1> ive tested the apk and helped valldrac test it to make the demo app usable

15:34:29 <r4v3r23[m]1> this CCS is important for the Android monero ecosystem. lets merge it

15:34:54 <plowsof11> thank you r4v3r23 - i also agree on merge (as do the updooters on the ccs)

15:35:17 <r4v3r23[m]1> where is the negative feedback coming from?

15:35:23 <luigi1111> :)

15:35:59 <r4v3r23[m]1> merge it and let valldrac, let the market decided if to fud, and have valldrac get to
work

15:36:06 <r4v3r23[m]1> s/and let valldrac//

15:36:12 <r4v3r23[m]1> s/and let valldrac//, s/fud/fund/

15:36:53 <plowsof11> hi luigi1111

15:39:06 <r4v3r23[m]1> r4v3r23[m]1: plowsof? why are people trying to add extra requirements to the CCS for
approval? why does molly need to add monero support for this CCS to be approved?

15:41:17 <plowsof11> it has already been promised, after the development milestone - which 6 months was the
guestimate from valldrac, but has 9months before it expires* r4v3r23 are you going to hunt the negative people
down and threaten them?

15:41:24 <r4v3r23[m]1> this SDK also will also enable more competition in the monero mobile market space,
something sorely needed

15:42:02 <r4v3r23[m]1> plowsof11: no, but they havent expressed it on the CCS proposal so why shouldnt back
channel discussions influence the decision?

15:42:09 <r4v3r23[m]1> s/shouldnt/should/

15:42:28 <r4v3r23[m]1> if they arent willing to say it publically then it should be discarded

15:42:34 <ceetee[m]> Is the last Molly CCS paid out?

15:42:55 <Rucknium[m]> r4v3r23: It seems completely reasonable to me for this discussion to involve Molly's
track record of completing CCSes

15:43:35 <plowsof11> any feedback disclosed to me in private will come from myself (which isnt even the case
here) lets not pretend like there are no negative feelings toward this

15:43:52 <plowsof11> im saying merge but not trying to sweep everything under the rug

15:44:19 <Rucknium[m]> I'm slightly positive for merge

15:44:21 <r4v3r23[m]1> Rucknium[m]: the reason for the SDK CCS is to fill the need for Molly's monero
implementation. valldrac says so directly.

15:45:00 <r4v3r23[m]1> i get that molly has taken longer than poeple expect, but this proposal attempts to
make progress towards that end

15:45:36 <r4v3r23[m]1> plowsof11: private negative feedback shouldnt influence a public process. whatevers
peoples reservations are, should be made public. this reeks of gatekeeping

15:45:57 <r4v3r23[m]1> and if valldrac doesnt do the work, they dont get paid. so whats the problem here?

15:46:23 <Rucknium[m]> The problem is donors contribute money and then they dont get what they paid for

15:46:54 <r4v3r23[m]1> Rucknium[m]: there have been more ridiculous CCSs - looking at you, Haveno UI

15:46:56 <Rucknium[m]> Then the money is in limbo

15:47:31 <r4v3r23[m]1> valldrac has already done the najority of the work on the SDK, they are not asking for
funding to start from scratch. at least 80% is done

15:47:35 <r4v3r23[m]1> s/najority/majority/

15:47:49 <ceetee[m]> Because the first Molly CCS hit so many roadblocks, I'm not convinced that this second
CCS will be completed on time and on budget either. However in general I'm willing to gamble on programming
proposals, especially if it's rather low level code like this. The potential benefits outweigh the risks in
this case

15:48:19 <r4v3r23[m]1> absolutely. this CCS will benefit the ecosystem as a whole. not jut molly

15:49:53 <r4v3r23[m]1> "We've accomplished 80% of the development of the Android library for Molly's use case"
<- low-risk for non-completion once funded

15:50:26 <plowsof11> merge it

15:50:51 <ceetee[m]> r4v3r23[m]1: the last 20% always take the longest in every project

15:51:34 <r4v3r23[m]1> ceetee[m]: itll get done. valldrac has the incentive to finish so they can move on to
molly

15:51:44 <r4v3r23[m]1> so yeah, merge it

15:52:47 <plowsof11> moving on (the outreach repurpose of funds is on-hold / and i assume will be changed) so
lets skip and move on to   d. [jeffro256 full-time dev 2023Q3](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/390)

15:52:56 <ofrnxmr[m]> Merge it plowluig

15:52:59 <plowsof11> a clear merge

15:53:16 <ofrnxmr[m]> Sdk n jeff

15:54:42 <ofrnxmr[m]> Greetings

15:54:48 <plowsof11> hello

15:55:38 <Rucknium[m]> Not having a clear idea of exactly what wallet2 does when it selects decoys has been a
big problem. jeffro says he will work on that in this CCS

15:55:39 <ofrnxmr[m]> Any opposed to jeffro? We need work to keep moving.

15:56:06 <ofrnxmr[m]> Easy merge. Anything ekse?

15:56:22 <plowsof11> in summary: merge everything (but i assume sdk and jeff need to adjust rates and what
not) jeffro256 also fix the typo on 4 months to 3

15:56:36 <ofrnxmr[m]> yes jeff

15:56:44 <ofrnxmr[m]> Fix typo

15:57:32 <plowsof11> hopefully ajs_ has not forgot about the out#reach funds and will update the plan for
their usage for us to discuss at a later date.

15:57:35 <ofrnxmr[m]> did ajs adenda item come up?

15:58:17 <plowsof11> on hold

15:58:21 <ofrnxmr[m]> Nono

15:58:27 <r4v3r23[m]1> LFG

15:58:39 <ofrnxmr[m]> Ummm "plowsof can you ass to agenda"

15:58:39 <ajs_[m]> I haven't a chance to work on this, busy with monerokon stuff.. will try to get to it for
next week

15:58:41 <ofrnxmr[m]> Add

15:58:55 <ofrnxmr[m]> Somthing like that i remenber reading

15:58:57 <plowsof11> ah oh thanks ajs_

15:59:15 <plowsof11> yes, if anyone is wanting to show/sell something for Monero at monerokon - they have
tables available for you!

15:59:25 <ajs_[m]> yes, please

15:59:29 <ajs_[m]> we have the space

15:59:49 <ajs_[m]> DM me if you would like a table

16:00:27 <ofrnxmr[m]> Thanks 🙏

16:00:32 <plowsof11> i would like to sell a dinner date with plowsof to McDonalds around the corner, not yet
finalised everything yet

16:01:26 <plowsof11> thanks all for attending !


```
Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
